pkg/resources/templates/config/login_identity_providers.go

```diff
@@ -68,9 +68,9 @@ var LoginIdentityProvidersTemplate = `<?xml version="1.0" encoding="UTF-8" stand
     <provider>
         <identifier>ldap-provider</identifier>
         <class>org.apache.nifi.ldap.LdapProvider</class>
-        <property name="Authentication Strategy">START_TLS</property>
```

Comment:
Introducing these configurations without defaults could break users who deployed this previously and benefitted from "Authentication Stategy"="START_TLS" and "Identity Strategy"="USE_DN". I'll recommend some changes